next/src/components/_global/newsletterSection/newsletterSection.module.scss

```diff
@@ -1 +1,118 @@
-.wrapper{}
\ No newline at end of file
+.newsletterSection {
+  display: grid;
+  grid-template-columns: 1fr 1.1fr;
+  align-items: flex-start;
+  grid-template-areas: "content visuals";
+
+  .content {
+    display: grid;
+    row-gap: 30px;
+    grid-area: content;
+
+    header {
+      display: grid;
+      grid-gap: 14px;
+    }
+
+    .description {
+      display: grid;
+      grid-gap: 16px;
+    }
+
+    .form {
+      display: grid;
+      gap: 20px;
+
+      >button {
+        margin-top: 25px;
+      }
+    }
+  }
+
+  .visuals {
+    position: relative;
+    grid-area: visuals;
+
+    .card {
+      position: relative;
+      display: grid;
+      grid-template-columns: auto 1fr;
+      grid-template-areas: "img heading" "img description";
+      align-items: center;
+      background: var(--background);
+      column-gap: 14px;
+      max-width: 500px;
+      border-radius: 16px;
+      box-shadow: 3px 2px 8px 0px rgba(143, 143, 143, 0.10),
+        13px 7px 15px 0px rgba(143, 143, 143, 0.09);
+      padding: 14px 40px 16px 12px;
+      margin-bottom: -40px;
+
+      .image {
+        width: 78px;
+        height: 78px;
```

Comment:
I suppose that there should be a clamp function to suit responsive design. Pay attention to responsiveness.